### PR TITLE
Added Properties: 'id' and 'className'

### PR DESCRIPTION
**Pull request by julionadal**
This two properties are missing from this types

### CODE REVIEW

File: types/components/Collapse.d.ts [hunk 1]
```diff
@@ -4,6 +4,8 @@ import { TransitionCallbacks } from './helpers';
 export interface CollapseProps
   extends TransitionCallbacks,
     React.ClassAttributes<Collapse> {
+  id?: string;
+  className?: string
```
Comment (jquense) [inline types/components/Collapse.d.ts]: they don't belong on this type Collapse, they belong on NavbarCollapse, and then they shouldn't be listed individually they should extend the react div props

Comment (jquense) [reply types/components/Collapse.d.ts]: see #2785

Comment (prateekkathal) [reply types/components/Collapse.d.ts]: Just FYI,

@julionadal A huge discussion was done on [this thread](https://github.com/react-bootstrap/react-bootstrap/pull/3448/commits/57278de982fb03e9b2e55a305f6f28379810706f#r256593836) in regards to the same issue. The decision to rebuild the API was made by @jquense and @thejohnfreeman

Comment (prateekkathal) [reply types/components/Collapse.d.ts]: Also this is a duplicate of #3502

Comment (julionadal) [reply types/components/Collapse.d.ts]: Thank you, i just saw @prateekkathal. Hope this changes are going to be released soon.

**Pull request closed without merging**